.travis.yml

```diff
@@ -35,6 +35,11 @@ matrix:
     - env: PGVERSION=13 TEST=ssl
     - env: PGVERSION=14 TEST=ssl
     - env: LINTING=true
+  allow_failures:
+    - env: PGVERSION=14 TEST=multi
+    - env: PGVERSION=14 TEST=single
+    - env: PGVERSION=14 TEST=monitor
+    - env: PGVERSION=14 TEST=ssl
```

Comment:
I think this can be removed now that your other PR is merged.

Reply:
I think we should keep Postgres 14 as allowed to fail builds/tests until it's released as 14.0. It's nice to follow along and fix when a beta introduces incompatibilities, though as we've seen here, as Travis installs Postgres from the community repositories, this may happen anytime in any PR we're working on... I feel like this should not block us.